### ISSUE TITLE
Usage fails with "Rollbar.init is not a function"

### ISSUE THREAD

**Issue by janpio** [NextFaze/angular-rollbar#9]
I think I followed the instructions with my Angular CLI project, but am getting these errors in the console:

```
zone.js:645 Error: Uncaught (in promise): TypeError: Rollbar.init is not a function
TypeError: Rollbar.init is not a function
    at new RollbarService (rollbar.service.js:23)
    at _createClass (core.es5.js:9601)
    at _createProviderInstance$1 (core.es5.js:9573)
    at initNgModule (core.es5.js:9527)
    at new NgModuleRef_ (core.es5.js:10635)
    at createNgModuleRef (core.es5.js:10619)
    at Object.debugCreateNgModuleRef [as createNgModuleRef] (core.es5.js:12903)
    at NgModuleFactory_.webpackJsonp.../../../core/@angular/core.es5.js.NgModuleFactory_.create (core.es5.js:13898)
    at core.es5.js:4556
    at ZoneDelegate.webpackJsonp.../../../../zone.js/dist/zone.js.ZoneDelegate.invoke (zone.js:391)
    at new RollbarService (rollbar.service.js:23)
    at _createClass (core.es5.js:9601)
    at _createProviderInstance$1 (core.es5.js:9573)
    at initNgModule (core.es5.js:9527)
    at new NgModuleRef_ (core.es5.js:10635)
    at createNgModuleRef (core.es5.js:10619)
    at Object.debugCreateNgModuleRef [as createNgModuleRef] (core.es5.js:12903)
    at NgModuleFactory_.webpackJsonp.../../../core/@angular/core.es5.js.NgModuleFactory_.create (core.es5.js:13898)
    at core.es5.js:4556
    at ZoneDelegate.webpackJsonp.../../../../zone.js/dist/zone.js.ZoneDelegate.invoke (zone.js:391)
    at resolvePromise (zone.js:770)
    at zone.js:821
    at ZoneDelegate.webpackJsonp.../../../../zone.js/dist/zone.js.ZoneDelegate.invokeTask (zone.js:424)
    at Zone.webpackJsonp.../../../../zone.js/dist/zone.js.Zone.runTask (zone.js:191)
    at drainMicroTaskQueue (zone.js:584)
    at <anonymous>
api.onUnhandledError @ zone.js:645
handleUnhandledRejection @ zone.js:667
_loop_1 @ zone.js:658
api.microtaskDrainDone @ zone.js:662
drainMicroTaskQueue @ zone.js:592
Error: Uncaught (in promise): TypeError: Rollbar.init is not a function
TypeError: Rollbar.init is not a function
    at new RollbarService (rollbar.service.js:23)
    at _createClass (core.es5.js:9601)
    at _createProviderInstance$1 (core.es5.js:9573)
    at initNgModule (core.es5.js:9527)
    at new NgModuleRef_ (core.es5.js:10635)
    at createNgModuleRef (core.es5.js:10619)
    at Object.debugCreateNgModuleRef [as createNgModuleRef] (core.es5.js:12903)
    at NgModuleFactory_.webpackJsonp.../../../core/@angular/core.es5.js.NgModuleFactory_.create (core.es5.js:13898)
    at core.es5.js:4556
    at ZoneDelegate.webpackJsonp.../../../../zone.js/dist/zone.js.ZoneDelegate.invoke (zone.js:391)
    at new RollbarService (rollbar.service.js:23)
    at _createClass (core.es5.js:9601)
    at _createProviderInstance$1 (core.es5.js:9573)
    at initNgModule (core.es5.js:9527)
    at new NgModuleRef_ (core.es5.js:10635)
    at createNgModuleRef (core.es5.js:10619)
    at Object.debugCreateNgModuleRef [as createNgModuleRef] (core.es5.js:12903)
    at NgModuleFactory_.webpackJsonp.../../../core/@angular/core.es5.js.NgModuleFactory_.create (core.es5.js:13898)
    at core.es5.js:4556
    at ZoneDelegate.webpackJsonp.../../../../zone.js/dist/zone.js.ZoneDelegate.invoke (zone.js:391)
    at resolvePromise (zone.js:770)
    at zone.js:821
    at ZoneDelegate.webpackJsonp.../../../../zone.js/dist/zone.js.ZoneDelegate.invokeTask (zone.js:424)
    at Zone.webpackJsonp.../../../../zone.js/dist/zone.js.Zone.runTask (zone.js:191)
    at drainMicroTaskQueue (zone.js:584)
    at <anonymous>
api.onUnhandledError @ zone.js:645
handleUnhandledRejection @ zone.js:667
_loop_1 @ zone.js:658
api.microtaskDrainDone @ zone.js:662
drainMicroTaskQueue @ zone.js:592
```

I did

1. run `npm install angular-rollbar --save`
2. add `import { RollbarModule, RollbarService } from 'angular-rollbar';` to the app.module
3. add `RollbarModule.forRoot({   accessToken: 'xxx' })` to the `imports` in the app.module
4. add `{ provide: ErrorHandler, useClass: RollbarService }` to the `providers` in the app.module
5. add `import { RollbarService } from 'angular-rollbar';` to the app.component
6. add `private rollbar: RollbarService` to the constructor of the app.component
7. call `rollbar.info('Logging to Rollbar!');` inside the constructor of the app.component

Did I miss anything?

(Sorry if I missed something simple, Angular noob)

**Comment by janpio** [comment]
[0.0.3](https://github.com/NextFaze/angular-rollbar/tree/v0.0.3) seems to be working, still I of course would like to be on the newest version.

**Comment by MichaelMarner** [comment]
Ugh, I'm really sorry I managed to let this one slip through. Fixed in v0.1.1.
